pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -7,6 +7,46 @@
             "#import": [ "base" ]
         },
 
+        "android": {
+            "#import": [ "any" ]
+        },
+        "android-arm64": {
+	           "#import": [ "any" ]
+        },
+        "android-mips": {
```

Comment:
Correct, no work has been done for MIPS that I'm aware of. (Some) work has been done for ARM, so I'd suggest replacing MIPS with arm.